core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -816,7 +821,7 @@ public PropServerConfiguration(
                     httpMinBindPort = p;
                 });
 
-                this.httpMinNetConnectionLimit = getInt(properties, env, PropertyKey.HTTP_MIN_NET_CONNECTION_LIMIT, 4);
+                this.httpMinNetConnectionLimit = getInt(properties, env, PropertyKey.HTTP_MIN_NET_CONNECTION_LIMIT, 32);
```

Comment:
nit: we set it to 64 in cloud that solved the issue, there is no downside for the higher value.

Reply:
ok, done